src/main/java/mate/academy/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -29,6 +29,8 @@ public void addSession(MovieSession movieSession, User user) {
         ShoppingCart shoppingCart = shoppingCartDao.getByUser(user);
         shoppingCart.getTickets().add(ticketDao.add(newTicket));
         shoppingCartDao.update(shoppingCart);
+        shoppingCart.getTickets().clear();
+        shoppingCartDao.update(shoppingCart);
```

Comment:
What will happened if we clear shopping cart every time, when user want to add ticket to the cart?